Feat/35 cron schedule

leewoojin12 opened the pull request:
---
name: "Pull Request"
about: "새로운 Pull Request 생성 시 사용합니다"
title: "[FEAT]/[FIX]/[DOC]/[STYLE]/[REFACTOR]: 간략한 설명"
labels: []
assignees: []
---


**1. 변경 요약 (Summary of Changes)**
- 이번 PR에서 변경된 주요 내용 요약

**2. 테스트 방법 (How to Test)**
- 로컬 환경에서 재현 방법:
    1. ...
- 테스트 시나리오:
    - 단위 테스트: ...
    - 통합 테스트: ...
    - e2e 테스트: ...

**3. 관련 이슈 (Related Issues)**
- 연결된 이슈 번호: `#이슈번호`

**4. 체크리스트 (Checklist)**
- [ ] 코드 컨벤션을 준수했는가? (파일명, 네이밍, 포맷 등)
- [ ] 새로운 기능/버그 수정에 대한 단위 테스트 작성 및 통과했는가?
- [ ] 통합 테스트 및 시나리오 테스트를 통과했는가?
- [ ] 문서(Docs) 업데이트가 필요한 경우 반영했는가?
- [ ] 주요 로직에 적절한 주석이 포함되었는가?
- [ ] 보안/성능 고려 사항 검토했는가?
- [ ] 관련 브랜치 전략 및 머지 대상이 적절한가?

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능
- 스케줄 셋팅은 되는데 필요없는 코드 날리기전

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능
- 스케줄 셋팅은 되는데 필요없는 코드 날리기전

- 필요 없는 코드 삭제 , 에러 코드 추가

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능
- 스케줄 셋팅은 되는데 필요없는 코드 날리기전

- 필요 없는 코드 삭제 , 에러 코드 추가

- 서비스 에러 처리 완

leewoojin12 committed:
Merge remote-tracking branch 'origin/develop' into feat/35-Cron_schedule

# Conflicts:
#	backend/src/main/java/com/example/backend/exception/ErrorCode.java

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능
- 스케줄 셋팅은 되는데 필요없는 코드 날리기전

- 필요 없는 코드 삭제 , 에러 코드 추가

- 서비스 에러 처리 완

chan0o0seo commented on backend/src/main/java/com/example/backend/build/service/BuildService.java:
```diff
@@ -1,230 +1,191 @@
+    private String jenkinsUrl1;
 
+    @Value("${JENKINS_API_TOKEN}")
+    private String apiToken1;
```
url이랑 api_token을 환경변수에서 받아오는게 아니라 jenkins info 에서 뽑아와서 써야해요

leewoojin12 committed:
[feat] cron 설정

- 파이프라인 생성되어 있는곳은 시간 설정 가능
- 스케줄 셋팅은 되는데 필요없는 코드 날리기전

- 필요 없는 코드 삭제 , 에러 코드 추가

- 서비스 에러 처리 완

- url , key  환경변수 ->jenkins info 에서 받아오는걸로 바꿈
